src/Microsoft.AspNetCore.Mvc.ViewFeatures/Buffers/PagedBufferedTextWriter.cs

```diff
@@ -9,7 +9,7 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace Microsoft.AspNetCore.Mvc.ViewFeatures.Internal
+namespace Microsoft.AspNetCore.Mvc.ViewFeatures.Buffers
 {
     public class PagedBufferedTextWriter : TextWriter
```

Comment:
Could `IViewBufferScope.CreateWriter(...)` return a `TextWriter`? That should allow this type to be `internal`.

Reply:
Yeah, that seems like a reasonable design. Added it as a note to the API review issue